src/fat/image.rs

```diff
@@ -73,6 +74,19 @@ impl Image {
         Ok(())
     }
 
+    pub fn bios_parameter(&self) -> BIOSParam {
+        let params: BIOSParam;
+        let mut bios_bytes: [u8; 13] = [0; 13];
+        bios_bytes.clone_from_slice(&self.boot_sector[11..24]);
+        params = unsafe {
+            let mut param = ::std::mem::zeroed();
+            let p = &mut param as *mut BIOSParam as *mut [u8; 13];
+            ::std::ptr::write(p, bios_bytes);
+            param
+        };
```

Comment:
Is this defined behavior? The compiler might reorder the struct's fields. As annoying as it is, prob should be done by hand.